src/tests/functional/plugin/gna/shared_tests_instances/behavior/ov_executable_network/exec_graph_info.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "behavior/ov_executable_network/exec_graph_info.hpp"
+
+#include "ie_plugin_config.hpp"
+#include <common_test_utils/test_constants.hpp>
+
+using namespace ov::test::behavior;
+namespace {
+const std::vector<ov::element::Type_t> netPrecisions = {
+        ov::element::i16,
+        ov::element::u8,
+        ov::element::f32
+};
+const std::vector<std::map<std::string, std::string>> configs = {
+        {},
```

Comment:
why is config empty? Shouldn't we check the both, GNA_SW_EXACT and GNA_SW_FP32 modes?

Reply:
This test does not compare the graph calculation, it checks the ability to use tensor names, but I added the  GNA_SW_EXACT mode